CONTRIBUTING.md

```diff
@@ -18,8 +25,64 @@ A post-install action is thrown. This action first install dependencies with `le
 
 At this point you are ready to contribute.
 
+## Storybook
+
+This tool will help you display component in a page where it will be isolated (rather than integrated in a page with many components).
+
+We are using the state of the art tool: [Storybook](https://storybook.js.org/).
+
+To launch it, you can run:
+
+```sh
+npm run storybook
+```
+
+## Keeping your forked repository up-to-date
+
+Before you start developping, it is important to always keep your forked repository **up-to-date**.
+
+### Link your forked repository to the original project
+
+For this, you will add a remote which points to the original repository (this has to be done only once):
+
+```sh
+git remote add upstream https://github.com/AxaGuilDEv/react-toolkit
+```
+
+### Update your master branch
+
+Then, each time you want to update your repository with the last changes, run:
+
+```sh
+git checkout master
+git fetch upstream
+git rebase upstream/master
+```
+
+More information in github documentation: [Fork a repo#Keep your fork synced](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo#keep-your-fork-synced)
+
 ## How to work on a component
 
+### Architecture of a component
+
+Here's an example in JS:
+```text
+├── dist/ : contains the generated JS/CSS files (in the postinstall process).
+├── src/
+    ├── __snapshots__ : generated storybook snapshots used to test the component.
+    ├── GreatComponent.js : the react component code.
+    ├── GreatComponent.md : more documentation about the component (props, ...).
```

Comment:
Sauf erreur de ma part mais il n' y a que des README.md

Reply:
Well... 9 occurrences... of such. 
So what is what's the proper way to do? :)

- https://github.com/AxaGuilDEv/react-toolkit/blob/master/packages/Form/Input/date/src/CustomDate.md
- https://github.com/AxaGuilDEv/react-toolkit/blob/master/packages/Form/Input/date/src/InputDate.md
- https://github.com/AxaGuilDEv/react-toolkit/blob/master/packages/Form/Input/choice/src/Choice.md
- https://github.com/AxaGuilDEv/react-toolkit/blob/master/packages/Form/Input/switch/src/Switch.md
- https://github.com/AxaGuilDEv/react-toolkit/blob/master/packages/Form/Input/date-phone/src/DatePhone.md
- https://github.com/AxaGuilDEv/react-toolkit/blob/master/packages/Form/Input/date-phone/src/InputDate.md
- https://github.com/AxaGuilDEv/react-toolkit/blob/master/packages/Layout/header/src/NavBar/NavBarItem.md
- https://github.com/AxaGuilDEv/react-toolkit/blob/master/packages/Layout/header/src/NavBar/NavBar.md
- https://github.com/AxaGuilDEv/react-toolkit/blob/master/packages/Layout/header/src/ToggleButton/ToggleButton.md